packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -156,7 +160,21 @@ export = (app: Application) => {
         return;
       }
 
-      const labels = issue.labels?.map(l => l.name);
+      let labels: string[] = [];
+      if (byConfig !== undefined) {
```

Comment:
Should we also check `context.payload.action == 'issues.opened'`?

Reply:
Good call, updated.